extra/xmlchecker.py

```diff
@@ -0,0 +1,43 @@
+import xmlschema
+import sys
+import xml.etree.ElementTree as ET
+import subprocess
+from pathlib import Path
+import os
+
+# change working directory in case script was not called from helios root directory
+survey_file = Path(sys.argv[1]).resolve()
+xsd_dir = Path(__file__).parent
+survey_file = str(survey_file.relative_to(xsd_dir.parent.resolve()))
+if Path.cwd() != xsd_dir.parent:
+    os.chdir(xsd_dir.parent)
+
+# check where helios.exe is located
+if any("helios.exe" in str(f) for f in Path.cwd().glob('*') if f.is_file()):
```

Comment:
should also support linux --> there the executable is just calles `helios`, not `helios.exe`

Reply:
why not try `.glob('helios*')` ?